### PR TITLE
[@types/forge-apis] Update to 0.7.2

### PR DESCRIPTION
**Pull request by dukedhx**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://www.npmjs.com/package/forge-apis
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by dukedhx** [comment]
Hi @andrewbranch @weswigham sorry to bother but can you spare a sec to review and approve this PR? Due to our limitations no one is really under the Autodesk-Forge organization that could approve as owner and I am now with Autodesk Forge to maintain these definitions. Thanks!

File: types/forge-apis/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for Forge-apis 0.4
+// Type definitions for Forge-apis 0.7
 // Project: https://github.com/Autodesk-Forge/forge-api-nodejs-client
 // Definitions by: Autodesk Forge Partner Development <https://github.com/Autodesk-Forge>
```
Comment (andrewbranch) [inline types/forge-apis/index.d.ts]: You should add your name and account link on a new line here

Comment (dukedhx) [reply types/forge-apis/index.d.ts]: done

Comment (andrewbranch) [reply types/forge-apis/index.d.ts]: Sorry, check the format against another package:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ebc4c80b7dc824d0e9d864b076dde5a93bae07a2/types/react/index.d.ts#L3-L24

I’m not sure whether our infrastructure understands it with a comma separator on a single line, but I’m guessing not.

Comment (dukedhx) [reply types/forge-apis/index.d.ts]: Yes comma separated accounts can be understood as per [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/forge-viewer/index.d.ts) - you can see Alan is being recognized as an owner.

**Review by andrewbranch — APPROVED** [review]

**Pull request merged**